nwb_conversion_tools/json_schema_utils.py

```diff
@@ -53,7 +54,9 @@ def get_schema_from_method_signature(class_method, exclude=None):
 
     """
     if exclude is None:
-        exclude = []
+        exclude = ['self']
+    else:
+        exclude = exclude + ['self']
```

Comment:
What is the advantage to doing it this way?

Reply:
Just clearer that it's always included by default, imo. Ultimately it's always included in the` exclude` list when it is invoked, but if someone looked at the code for the first time they'd have to examine the interiors of the loop to see how it always skips `self`.